mkdocs/docs/user_guide.md

```diff
@@ -2708,7 +2710,7 @@ This visualization produces a pie chart comparing the predictions of the two mod
 
 ### compare_classifiers_predictions_distribution
 
-This visualization uses the `ground_truth`, `ground_truth_metadata`, `field`, `predictions` and `model_names` parameters.
+This visualization uses the `ground_truth`, `ground_truth`, `field`, `predictions` and `model_names` parameters.
```

Comment:
double check if this is correct, ground truth is repeted twice

Reply:
This was the old declration of the function - no metadata used:

def compare_classifiers_predictions_distribution(
        predictions,
        ground_truth,
        field,
        labels_limit,
        model_names=None,
        output_directory=None,
        file_format='pdf',
        **kwargs
):

I have removed the repeated word.